casestudies/load-testing/README.md

```diff
@@ -0,0 +1,70 @@
+# ACK Performance at Scale
```

Comment:
@a-hilaly not clear where and how the multi-line string you are referring to should be used.

Reply:
In markdown you do can something like:
```markdown
This
is a multi line
text that will be rendered as a single line.
It helps to keep the markdown files clean and readable,
and it's easier to submit reviews that target a
specific line/context.
```
and it will be rendered to:
```
This is a multi line text that will be rendered as a single line. It helps to keep the markdown files clean and readable, and it's easier to submit reviews that target a specific line/context.
```

Reply:
from https://www.dotcms.com/docs/latest/markdown-syntax#Paragraphs
> Note: If you don't leave a blank line between blocks of text, they will be collapsed into a single paragraph.

Reply:
Updated the same in the latest commit